### ISSUE TITLE
(accessibility) User status is not announced. User listening, user talking.

### ISSUE THREAD

**Issue by amirmahdifard0** [BearWare/TeamTalk5#853]
When we scroll through the list of members of a channel, the screen reeder does not read to us what this user is currently up to. Is talking or listening. some notes. This problem only exists for the Windows teamtalk and does not exist for the Android application. The second point. I only tested this with nvda screen reeder and I do not know if other screen reeders like jaws have this problem or not.

**Comment by Oreonan** [comment]
- For classic client: you have to press ctrl-g to hear if an user is currently talking
- For QT client: if an user is talking, a 🎤 emoji is present after his nickname

**Comment by amirmahdifard0** [comment]
> * For classic client: you have to press ctrl-g to hear if an user is currently talking
> * For QT client: if an user is talking, a 🎤 emoji is present after his nickname

If it announces it on the usernames, it's better and makes things faster.

**Comment by amirmahdifard0** [comment]
If it announces it on the usernames, it's better and makes things faster.

**Comment by amirmahdifard0** [comment]
Will you do that?

**Comment by amirmahdifard0** [comment]
How can I download the teamtalk translation string file?

**Comment by bear101** [comment]
You can download the "raw" Android version here
<https://raw.githubusercontent.com/BearWare/TeamTalk5/master/Client/TeamTalkAndroid/src/main/res/values/strings.xml>.

More on translations here <https://github.com/BearWare/TeamTalk5/wiki>.

--
Kind regards

Bjørn Damstedt Rasmussen
www.bearware.dk
